kzz7yh-f72787/cod-ve09v2_kzz7yh-f72787.xml

```diff
@@ -587,8 +591,8 @@
       inferiori<lb ed="#V" n="114" break="no"/>portione rationis. 
     </p>
                <p xml:id="kzz7yh-f72787-d1e1071">
-                  <lb ed="#V" n="115"/>Ad primum argumentum ad pirte aliam 
-      pri<lb ed="#V" n="116" break="no"/>quod solus consensus in delectaione 
+                  <lb ed="#V" n="115"/>Ad primum argumentum ad parte aliam dicendum
+      <lb ed="#V" n="116"/>quod solus consensus in delectaione 
```

Comment:
```suggestion
      <lb ed="#V" n="116"/>quod solus consensus in <sic>delectaione</sic> 
```

Reply:
It is misspelled in the manuscript, so you were right to not change it, but I think we should mark that it is spelled incorrectly.